lib/request_handler/body_parser.rb

```diff
@@ -2,76 +2,35 @@
 
```

Comment:
I agree if we remove BodyParser the base class would need to know how to read the request body from a request.